subtrack.MAUI/Utilities/SubscriptionMapper.cs

```diff
@@ -1,13 +1,16 @@
 ﻿using subtrack.DAL.Entities;
+using subtrack.MAUI.DateAndTimeProvider;
 using subtrack.MAUI.Responses;
 using subtrack.MAUI.Services;
 
 namespace subtrack.MAUI.Utilities
 {
     public static class SubscriptionMapper
     {
-        public static IEnumerable<SubscriptionResponse> ToResponses(this IEnumerable<Subscription> subscriptions)
+        public static IEnumerable<SubscriptionResponse> ToResponses(this IEnumerable<Subscription> subscriptions, IDateTimeProvider date)
         {
+            var calculator = new SubscriptionsCalculator(date);
```

Comment:
I would prefer if subscriptionscalculator is registered as a service and then injected everywhere instead of manually creating a new instance, in case it gets more dependencies in the future. Static methods like this one can ofc take the subscriptionscalculator as a parameter